web/satellite/.env

```diff
@@ -1,2 +0,0 @@
-VUE_APP_STRIPE_PUBLIC_KEY=pk_test
```

Comment:
We need the .env file just not the real value so that people know what values to put in there when we deploy a satellite